src/.editorconfig

```diff
@@ -170,6 +170,7 @@ indent_size = 2
 end_of_line = lf
 
 [*.cs]
+dotnet_diagnostic.IDE0005.severity = error
```

Comment:
Not a big fan of erroring out because of non-functional problems. We often make stuff like this (extra usings) a warning, and in CI build we use warnAsError